SparkPost/2web2ui
# FE-1291 - Upgrade Hibana Matchbox to version 4.7.2 and styled components to >= 5.2.0

[FE-1291](https://sparkpost.atlassian.net/browse/FE-1291)

### What Changed

- Upgrades Matchbox to version 4.7.2 for Hibana only
- Updates Styled Components to the latest version (this is a peer dependency)
- Updates the custom `Panel.Headline` component, removing HTML semantics now that the `Panel.Header` supplies those semantics as expected

### How To Test

- Verify the Dashboard visually looks the same as it does in production
- Look at the Matchbox release notes (see the Jira ticket) and verify that breaking changes were not introduced

### To Do
